spec/acceptance/letsencrypt_plugin_dns_cloudflare_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'letsencrypt::plugin::dns_cloudflare' do
+  supported = case fact('os.family')
+              when 'Debian'
+                # Debian 9 has it in backports, Ubuntu started shipping in Bionic
```

Comment:
Debian 9 and Ubuntu 16.04 are no longer in metadata.json so they're no longer tested. I think this whole case can be simplified

Reply:
Case reworked and obsolete content removed